packages/playground/Samples/nativeLayout.tsx

```diff
@@ -3,56 +3,72 @@
  * Licensed under the MIT License.
  * @format
  */
-import React from 'react';
+import React, {useState} from 'react';
 import {AppRegistry, View, Text, requireNativeComponent} from 'react-native';
 
 const GridView: any = requireNativeComponent('PlaygroundGridView');
 const GridItemView: any = requireNativeComponent('PlaygroundGridItemView');
 
 function Bootstrap() {
+  const [hover1, setHover1] = useState(false);
+  const [hover2, setHover2] = useState(false);
+  const [hover3, setHover3] = useState(false);
+  const [hover4, setHover4] = useState(false);
+
+  const getHoverProps = (setter: (value: boolean) => void) => {
+    return {
+      onMouseEnter: () => setter(true),
+      onMouseLeave: () => setter(false),
+    };
+  };
```

Comment:
Took a moment to realize _why_ you were doing this hover text changing. The reason I take it is that you have a XAML item (GridViewItem) with a Yoga-driven child that's changing layout size (due to the font change).

Adding a comment to describe the _why_ of this would help with the maintainability of the code.